src/index.tsx

```diff
@@ -1,4 +1,4 @@
 import { createRoot } from 'react-dom/client';
```

Comment:
All strings should be double quoted based on the `@mate-academy/stylelint-config` style config, so i won't be referring to this particular rule on the review from now on, the CLI command `npm run lint` should fix this issue (and others like this) across the whole project.